src/frontend/src/lib/derived/address.derived.ts

```diff
@@ -9,6 +9,11 @@ export const addressNotLoaded: Readable<boolean> = derived([addressStore], ([$ad
 	isNullish($addressStore)
 );
 
+const ethAddressData: Readable<AddressData | null | undefined> = derived(
```

Comment:
Do we maybe already have a type that equals `AddressData | null | undefined`?

Reply:
Included